docs/source/pipeline.rst

```diff
@@ -0,0 +1,270 @@
+               name: str_count
+               params:
+                   pat: white
+       # 3. Create a new column with Method
+       # Note: Don't specify 'name' in params
+       col3:
+           transform_method:
+               name: cummax
+               params:
+                   column: fixed_acidity
+       # 4. Complex Sql
+       col4:
+           sql: fixed_acidity * density
+       # 5. Multiple Params
+       col5:
+           transform_method:
+               name: regexp
+               params:
+                   column: color
+                   pattern: "white"
+                   method: "replace"
+                   replacement: "NOT white"
+       # 6. Multi-Stage Transforms
+       col6:
+           sql: color
+           transform_method1:
+               name: str_count
+               params:
+                   pat: white
+           transform_method2:
+               name: add
+               params:
+                   x: 0.5
+       # 7. Using Previously Created Columns
+       col7:
+           sql: col2
+           transform_method:
+               name: add
+               params:
+                   x: 0.5
+
+Train
+~~~~~
+
+| `train functions
+  documentation <https://www.vertica.com/python/documentation/1.0.x/html/machine_learning_vertica.html#api-machine-learning-vertica>`__
+| If you want to train a model the default setting is use all the
+  previously created **cols** as predictors to your method. If you wish
+  to exclude some columns use ``exclude:`` to subtract the specified
+  columns from the default or use ``include:`` to strictly choose the
+  subset. The previous transform example is the basis for this train
+  example:
+
+**Example 1**
+
+.. code:: yaml
+
+     train:    
+       method:
+           name: RandomForestClassifier
+           target: citric_acid
+           params:
+               n_estimators: 40
+               max_depth: 4
+
+**Example 2**
+
+.. code:: yaml
+
+     train:    
+       method:
+           name: LinearSVC
+           target: col2
+           exclude: ['col5', 'col2']
+
+**Example 3**
+
+.. code:: yaml
+
+     train:    
+       method:
+           name: LinearSVC
+           target: col2
+           include: ['col1', 'col3', 'col4', 'col6', 'col7']
+
+Test
+~~~~
+
+| You may want to compute metrics for your newly created model. To see a
+  list click
+  `here <https://www.vertica.com/python/documentation/1.0.x/html/machine_learning_metrics.html#api-machine-learning-metrics>`__.
+  The results are stored in
+| **``[schema].[pipeline_name]_METRIC_TABLE``**
+
+**Example 1**
+
+.. code:: yaml
+
+     test:
+       metric1: 
+           name: accuracy_score
+           y_true: quality
+           y_score: prediction
+       metric2: 
+           name: r2_score
+           y_true: quality
+           y_score: prediction
+       metric3: 
+           name: max_error
+           y_true: quality
+           y_score: prediction
+
+.. image:: _static/metric_table.png
```

Comment:
Same comment as the other table. We should have HTML tables rather than static images.